app/controllers/books_controller.rb

```diff
@@ -0,0 +1,44 @@
+class BooksController < ApplicationController
+  before_action :find_book, only: [:edit, :update, :destroy]
+
+  def index
+    @books = Book.ordered.page params[:page]
```

Comment:
I would say that if we are getting in the habit of having consistent use of bracing, we should also wrap `params[:page]` here

`@books = Book.ordered.page(params[:page])`